tox.ini

```diff
@@ -64,8 +62,10 @@ sitepackages = true
 # f'' is used inside docs
 basepython = python3
 deps =
-    flake8>=3.6.0,<4
-    yamllint>=1.15.0,<2
+    pre-commit>=1.15.2
+commands =
+    # to run a single linter you can do "pre-commit run flake8"
```

Comment:
Maybe one for the contributing documentation? People will most likely go there first ...

Reply:
Not exactly. The documentation should mention tox as in `tox -e pre-commit -- flake8`.
Anyway, this needs updating because of a merge conflict. So @ssbarnea should get a chance to rethink it.